bitacross-worker/litentry/core/direct-call/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+authors = ["Trust Computing GmbH <info@litentry.com>"]
+edition = "2021"
+name = "lc-direct-call"
+version = "0.1.0"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42", default-features = false }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42", default-features = false }
+
+# internal dependencies
+itp-stf-primitives = { path = "../../../core-primitives/stf-primitives", default-features = false }
+litentry-primitives = { path = "../../primitives", default-features = false }
+parentchain-primitives = { package = "core-primitives", path = "../../../../primitives/core", default-features = false }
+teerex-primitives = { path = "../../../../primitives/teerex", default-features = false }
```

Comment:
Shall we ditch `teerex-*` stuff now? Like use `itp_types` or `litentry_primitives` instead for `ShardIdentifier`

I mean there will be a clean-up, but we can try to not add new references

Reply:
Yes, I've removed this dep.